src/Relation/Nullary.agda

```diff
@@ -26,6 +26,10 @@ infix 2 _because_
 ¬_ : ∀ {ℓ} → Set ℓ → Set ℓ
 ¬ P = P → ⊥
 
+-- level polymorphic version of ¬
+¬′ : ∀ {ℓ} (b : Level) → Set ℓ → Set (ℓ ⊔ b)
+¬′ {_} b P = P → Lift b ⊥
```

Comment:
Hmm this name needs to change for two reasons. Firstly we're now using `\prime` instead of `'`. But more importantly, the convention is that `\prime` signifies something is less polymorphic, not more.

The fact that you have to provide the level as well is a bit upsetting...

Reply:
It occurs to me that my browser doesn't actually render the difference between `'` and `\prime` so apologies if it's already `\prime`!

Reply:
(it was already `\prime`). Yeah, I'm not happy with the name either, not that the level is manditory. I tried to make it implicit, and then had to actually give it too often.

`\neg\_p` ?

Reply:
Turns out that looks awful in the usual fonts. `¬ᵢ` for "`¬` indexed by a level" looks better.